docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -21,10 +21,18 @@ the destination platform from the [downloads page](https://dl.gitea.io/gitea), c
 the URL and replace the URL within the commands below:
 
 ```sh
-wget -O gitea https://dl.gitea.io/gitea/1.4.2/gitea-1.4.2-linux-amd64
+wget -O gitea https://dl.gitea.io/gitea/1.4.3/gitea-1.4.3-linux-amd64
 chmod +x gitea
 ```
 
+## Verify GPG signature
+Gitea signs all binaries with a [GPG key](http://pool.sks-keyservers.net/pks/lookup?op=get&search=0x2D9AE806EC1592E2) to prevent against unwanted modification of binaries. To validate the binary download the signature file which ends in `.asc` for the binary you downloaded and use the gpg command line tool.
```

Comment:
Please use the same server for verifying command and `GPG key` link:
[GPG key](https://pgp.mit.edu/pks/lookup?op=vindex&fingerprint=on&search=0x2D9AE806EC1592E2)